src/client/custom_blocks.js

```diff
@@ -25,6 +25,20 @@ Blockly.defineBlocksWithJsonArray([
     "nextStatement": null,
     "colour": 120,
   },
+  {
+    "type": "direct_move",
+    "message0": "Direct move %1",
```

Comment:
I think I understand what you want to do. Would it make sense to have the message "Always" to indicate that it is unconditional? Also, did you test whether it works with the existing backend?